Makefile

```
@@ -379,19 +379,9 @@ golangci_version=v1.51.2
 lint:
 	@echo "--> Running linter"
 	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
-	@$(golangci_lint_cmd) run --timeout=10m
+	@bash scripts/go-lint-all.sh
```

```suggestion
	@sh ./scripts/go-lint-all.sh
```

small nit for consistency with the other scripts